GameEngine/DeterministicDeck.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GameEngine
+{
+    public class DeterministicDeck : Deck
+    {
+        public List<CardType> cards { get; private set; }
```

Comment:
❔ Why is the getter public when we have `Cards` already?

Reply:
Oversight. Fixed.